pkg/apis/defaults/configmaps.go

```diff
@@ -9,4 +9,8 @@ const (
 	RadixGitChangedFolders = "git-changed-folders"
 	// RadixGitChangedChangedRadixConfig Indicates if Radix config file is changed in the GitHub source
 	RadixGitChangedChangedRadixConfig = "git-changed-radixconfig"
+	GitPublicKeyConfigMapName         = "git-ssh-public-key"
+	GitPublicKeyConfigMapKey          = "public-key"
```

Comment:
```suggestion
	GitPublicKeyConfigMapKey          = "public_key"
```
Use underscore to align with name of private key